CONDUCT.md

```diff
@@ -1,12 +1,94 @@
-# [<img src="https://avatars1.githubusercontent.com/u/3771091?v=3&s=30">](https://github.com/gonum) Working Together
+# [<img src="https://avatars1.githubusercontent.com/u/3771091?v=3&s=30">](https://github.com/gonum) Gonum Community Code of Conduct
 
-The Gonum project welcomes contributions from all. Contributors should foster an open and welcoming environment. When contributing or otherwise participating, please:
+## About
+
+Online communities include people from many different backgrounds. The Gonum contributors are committed to providing a friendly, safe and welcoming environment for all, regardless of gender identity and expression, sexual orientation, disabilities, neurodiversity, physical appearance, body size, ethnicity, nationality, race, age, religion, or similar personal characteristics.
+
+The first goal of the Code of Conduct is to specify a baseline standard of behavior so that people with different social values and communication styles can talk about Gonum effectively, productively, and respectfully.
+
+The second goal is to provide a mechanism for resolving conflicts in the community when they arise.
+
+The third goal of the Code of Conduct is to make our community welcoming to people from different backgrounds. Diversity is critical to the project; for Gonum to be successful, it needs contributors and users from all backgrounds. (See [Go, Open Source, Community.](https://blog.golang.org/open-source))
+
+We believe that healthy debate and disagreement are essential to a healthy project and community. However, it is never ok to be disrespectful. We value diverse opinions, but we value respectful behavior more.
```

Comment:
I wouldn't use "ok" in a formal document. It should be at least spelled "OK", or maybe changed to "acceptable" (though "OK" sounds best to me personally).